Core/src/main/java/com/intellectualcrafters/plot/PS.java

```diff
@@ -1961,6 +1961,31 @@ public PlotArea getPlotAreaByString(String search) {
             return null;
         }
     }
+    
+    /**
+     * Get Plots based on alias
+     *
+     * @param alias to search plots
+     * @param worldname to filter alias to a specific world [optional] null means all worlds
+     *
+     * @return Set<{@link Plot}> null if nothing found
```

Comment:
This should return an empty set rather than null. Always avoid using null when it is possible (that's why we :heart: Optional<>)

Reply:
okay normally i would do this too but i found a lot of code returning colletions null instead of empty so i thought this is your code style ;)

Reply:
@Opachl I know that the code base is a bit... messy. However, it is quite hard to change things at this point without losing backwards compatibility. I think it'd be smart to release a breaking version soon(~ish) and make the code pretty again. As you may have noticed, there code base also struggles with packages and whatnot.

So yeah, I see your point :laughing: But let's try to make things pretty moving forward.